### PR TITLE
update loadAudioFiles and doAuditoryMotion for the new sounds

### PR DESCRIPTION
**Pull request by marcobarilari**
still in WIP, it will break up things since the expDesign is not still ready to accommodate the new design (motionLRL and motion RLR). I will PR to this PR soon

### CODE REVIEW

**Comment by marcobarilari** [comment]
ok, made a bit of confusion... I cannot send a pr to this pr. You see all the commits that make these changes working here

**Comment by marcobarilari** [comment]
`equateRMSWav` is far away to be at a good refactoring level, it is there so that we can start working with the new sounds

File: initEnv.m [hunk 1]
```diff
@@ -102,5 +102,6 @@ function addDependencies()
     addpath(genpath(fullfile(pth, 'lib', 'CPP_BIDS', 'src')));
     addpath(genpath(fullfile(pth, 'lib', 'CPP_PTB', 'src')));
     addpath(fullfile(pth, 'subfun'));
+    addpath(fullfile(pth, 'input'));
```
Comment (Remi-Gau) [inline initEnv.m:105]:
```suggestion
```

Comment (Remi-Gau) [reply initEnv.m:105]: removing this as it seems to be not needed

File: input/equateRmsWav.m [hunk 2]
```diff
@@ -0,0 +1,79 @@
+function equateRmsWav(subjName)
+
```
Comment (Remi-Gau) [inline input/equateRmsWav.m:2]:
```suggestion
% function equateRmsWav(subjName)
% Not used during the experiment itself but allows to root mean square the sounds of a given subject.
```

File: input/equateRmsWav.m [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    targetWavFn = [subjName, '_LRL_T.wav'];
+    runFunction(referenceWavFn, targetWavFn);
+
+    targetWavFn = [subjName, '_RLR_T.wav'];
+    runFunction(referenceWavFn, targetWavFn);
+
+end
+
+function runFunction (referenceWavFn, targetWavFn)
```
Comment (Remi-Gau) [inline input/equateRmsWav.m:37]: This looks like a good candidate to take out of this and put in CPP_PTB. 

Not urgent can be done later.

**Review by Remi-Gau — CHANGES_REQUESTED** [review]
Only added a stub for a help section but that looks good to me.

**Review by Remi-Gau — APPROVED** [review]

**Pull request merged**